src/adr/0026-removing-configured-additional-mailboxListeners.md

```diff
@@ -19,17 +19,32 @@ binds to the main event exchange.
 If the user unconfigures the listener, the queue and the binding are still present but not consumed. This results in 
 unbounded queue growth eventually causing RabbitMQ resource exhaustion and failure.
 
+## Vocabulary
+
+A **required group** is a group configured within James additional mailbox listener or statically binded via Guice. We 
+should have a queue for that mailbox listener binded to the main exchange.
+
+A **registered group** is a group whose queue exist in rabbitMQ and is bound to the exchange, independently of its James 
+usage. If it is required, a consumer will consume the queue. Otherwise the queue might grow unbounded.
+
 ## Decision
 
 For rabbitMQ, configuration changes of additional mailbox listeners should be tracked via event sourcing. Event sourcing is 
```

Comment:
*R*abbitMQ

Reply:
I think it should be articulated in the other way: here is what properties we need... we have chosen ES because it is a appropriate solution to the above-listed issues.